### DISCUSSION TITLE
Convert text to path

### DISCUSSION THREAD

**Discussion by burnoutberni** [uxbox/uxbox#1964] [Q&A] [answered]
Quick question: Is it possible to convert text to path using Penpot? If so, how? :)

**Comment by elhombretecla** [comment, marked as answer]
Hi @burnoutberni :)

By the moment is not available to convert texts to path in Penpot but it is in our roadmap.

We are doing a lot of improvements with the path edition and we hope to start working on this asap.

Thanks!

**Reply by heyakyra** [reply]
Is it possible to convert the stroke of an object to a path?
